### ISSUE TITLE
Cannot access replace menu header icons with keyboard

### ISSUE THREAD

**Issue by barmac** [camunda/camunda-modeler#3437]
### Describe the bug

When I open a replace menu, I cannot access the header icons, e.g. ad-hoc subprocess.

### Steps to reproduce

https://user-images.githubusercontent.com/28307541/217776775-822f45d7-25ef-4bb3-9091-035eb1a8e5f4.mov

### Expected behavior

When I Shift-Tab from the first entry, I should focus the last header icon.

### Environment

- OS: [e.g. MacOS 10.2, Windows 10]
- Camunda Modeler Version: 5.8.0-rc.0
- Execution Platform: [e.g. Camunda Platform, Camunda Cloud]
- Installed plug-ins: [...]
